src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -134,26 +146,34 @@ func importConfig(
 		// Special case 400 (validation) errors with friendly output
 		// about which configuration properties were invalid.
 		if he.Response.StatusCode == 400 {
-			fmt.Fprint(os.Stderr, formatValidationError(err, he))
+			ve, err := formatValidationError(err, he)
+			if err != nil {
+				return fmt.Errorf("error setting config: %v", err)
+			}
+			fmt.Fprint(os.Stderr, ve)
 			out.Die("No changes were made.")
```

Comment:
How about merging these two lines into one?

Also, I think this exercise and the previous exercise with defer changes are showing me that we should move to have no output in functions, only in Run functions. It'll be easier to reason about, though it'll force some changes.

Something to consider for future PRs / changes?

Reply:
```
			ve, err := formatValidationError(err, he)
			if err != nil {
				return fmt.Errorf("error setting config: %v", err)
			}
			return fmt.Errorf("%s\nNo changes were made", ve)
```
I merged like this because L151 is an error that occurred during formatValidationError, the other one just prints and says that no changes were made 👍